tests/test_types_typeddict.py

```diff
@@ -192,16 +194,73 @@ class Data(BaseModel):
     class DataTD(TypedDict):
         a: int
 
+    class CustomTD(TypedDict):
+        b: int
+
+        @classmethod
+        def __get_pydantic_core_schema__(
+            cls, source_type: Any, handler: GetCoreSchemaHandler
+        ) -> core_schema.CoreSchema:
+            schema = handler(source_type)
+            schema['computed_fields'] = [
+                core_schema.computed_field(property_name='another_b', return_schema=core_schema.int_schema())
+            ]
+            return schema
+
     class Model(BaseModel):
         data: Data
         data_td: DataTD
+        custom_td: CustomTD
```

Comment:
Here I would also suggest separating cases. Consider this example of using classes as `parametrize` arguments https://github.com/pydantic/pydantic/blob/bb79d99ad3e93e33997e0b0eda23bc3c52919ace/tests/test_root_model.py#L9

Reply:
I'd have to call the fixture `TypedDict` directly if I want to do that - which I'm fine with the current one 👀 - but if you have suggestions...

Reply:
It's up to you. If it looks too difficult improving right now let's keep it as is then.